### PR TITLE
Feature/pro 1593/combined vehicle sd minting

### PR DESCRIPTION
**Pull request by LorranSutter**

### CODE REVIEW

File: contracts/implementations/MultipleMinter.sol [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+error DeviceAlreadyRegistered(address addr);
+error InvalidSdSignature();
+
+contract MultipleMinter is
+    AccessControlInternal,
+    VehicleInternal,
+    SyntheticDeviceInternal
+{
+    bytes32 private constant MINT_VEHICLE_SD_TYPEHASH =
+        keccak256("MintVehicleAndSdSign(uint256 integrationNode)");
```
Comment (elffjs) [inline contracts/implementations/MultipleMinter.sol:28]: We have
```
MintVehicleSign(uint256 manufacturerNode,address owner,string[] attributes,string[] infos)
```
does this seem a little thin, by comparison?

Comment (elffjs) [reply contracts/implementations/MultipleMinter.sol:28]: Ohh, the owner signs `MintVehicleSign` still? This is difficult to follow.

Comment (LorranSutter) [reply contracts/implementations/MultipleMinter.sol:28]: Yep, afterall the main thing here is still the vehicle minting I suppose.

**Review by elffjs — APPROVED** [review]

**Comment by elffjs** [comment]
Merge merge go go

**Pull request merged**